### DISCUSSION TITLE
Groups on home screen?

### DISCUSSION THREAD

**Discussion by KamilAlx** [tanujnotes/Olauncher#174] [Ideas]
Just wondering if anyone else think that groups/folders on home screen would be useful? Like social networks, games, finance and etc.

**Comment by tanujnotes** [comment]
I'm curious. Why do you need groups in a launcher like this?

**Reply by KamilAlx** [reply]
In order not to access menu list. Want to have easy access to many items on homescreen, while keeping it minimalistic.

**Reply by tanujnotes** [reply]
I'm sorry but that's not a good enough reason. And it's not minimal either IMHO. Even if groups are added, they'll appear in the app drawer and not on home screen.

**Comment by Dewet-sa** [comment]
But mybe be able to assess some shortcuts like website (chrome) added to home screen to appear on list app drawer list

**Reply by tanujnotes** [reply]
I'm not planning to work on this anytime soon but if someone sends a pull request, I'll be happy to add it.
